Gruntfile.js

```diff
@@ -79,7 +79,7 @@ module.exports = function( grunt ) {
 			},
 			test: {
 				src: [ "test/*.js", "test/functional/**/*.js", "test/unit/**/*.js",
-					"!test/config.js" ],
+					"test/compiler/**/*.js", "!test/config.js", "!test/compiler/_compiled/**" ],
```

Comment:
For the generated compiled thing, we can use the existing `tmp/`. For the tests, let's use the two separate directories for now. Possibly we could consolidate the test framework to be mocha and merge both directories back into one. For now, I want to avoid having mixed content, see the unnecessary jshintrc additions for qunit perspective.